### ISSUE TITLE
Floating window (combat timer) resetting its location.

### ISSUE THREAD

**Issue by AvroraAurora** [xorus/EngageTimer#70]
Currently having an issue where the combat timer (the one that shows how long I have been in combat) keeps going back to its default location (top left corner of my screen). I have the lock floating window option enabled but the timer always goes back to the top left corner after I close and reopen the game (logging out and in doesn't seem to reset the location). I'm not sure if this is intended or not.

**Comment by xorus** [comment]
That's a bit weird, do you have this problem with other plugins ? Do you resize your window often ?
The plugin does not have any control over the combat timer position (it only sets it to non-movable when you lock it) so this sounds more like a dalamud issue. 
Does the window go back to its original position every time you open the game ?

Looking at the xivlauncher discord I saw this comment by someone for chat2 :
![image](https://github.com/user-attachments/assets/1792126e-82c5-43e2-88f7-f4290256f05f)

(https://discord.com/channels/581875019861328007/1224865018789761126/1272995290240253972)

But I couldn't find if they asked about it in the mentioned channels. Maybe for this you could try deleting the global window position save file, I'm not sure in which directory it is but should be somewhere in APPDATA > xivlauncher. The file is named `dalamudUI.ini`, be aware that it will reset all your window positions. (under linux it's in `.xlcore`)

**Comment by xorus** [comment]
(the game must be closed before modifying the file)

**Comment by AvroraAurora** [comment]
Just tried the delete dalamudUI.ini method and it didn't fix the issue. This might actually be a Dalamud issue now that your bring it up. There are a few plugins that just refuses to have their UI location saved. Specifically, the ones that lets me "unlock" and "lock" the location of the UI element and letting me drag around freely. UI that uses the coordinates system are saved just fine.

**Comment by AvroraAurora** [comment]
Okay so I did a little bit of research and I think I found the reason why my UI doesn't save. 

The user Im logged into right now on my computer has special non-English characters (it is in Japanese). And I think this is one of those issues that makes it so that some program is unable to update or access the files in the appdata folder because of that. (not the first time I've had this problem, but I wasn't expecting dalamud to have it)
